dask_expr/_expr.py

```diff
@@ -2805,6 +2824,10 @@ def optimize(expr: Expr, fuse: bool = True) -> Expr:
         Input expression to optimize
     fuse:
         whether or not to turn on blockwise fusion
+    common_subplan_elimination : bool, default False
```

Comment:
Note that `common_subplan_elimination` does not mean _no_ CSE but rather _less_ CSE. We may want to reflect that in the docstring.